### PR TITLE
fix: Use per OS login command in quick start guide

### PR DESCRIPTION
**Pull request by erezrokah**
#### Summary

In our quick start guides we tell users to run `cloudquery login`. However we use the same command for all operating systems. `cloudquery login` won't work for Linux (unless added to the PATH) or Windows
image

This PR fixes the issue by using `cloudquery login` on macOS, `./cloudquery login` on Linux and `cloudquery.exe login` on Windows. The fix is consistent with how we tell users to run a sync a bit below in the guide:

#### macOS
image
#### Linux
image
#### Windows
image

### CODE REVIEW

File: website/components/mdx/_configure-linux.mdx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import ConfigureDestination from './_configure-destination.mdx';
 import ConfigureSource from './_configure-source.mdx';
 import ConfigureAuthenticateProvider from './_configure-authenticate-provider.mdx';
 import ConfigureStartSyncing from './_configure-start-syncing.mdx';
-import ConfigureSyncCommandUnixPrecompiled from './_configure-sync-command-unix-precompiled.mdx';
```
Comment (erezrokah) [inline website/components/mdx/_configure-linux.mdx:5]: Unrelated fix to make this filename more clear

File: website/components/mdx/_configure-macOS.mdx [hunk 2]
```diff
@@ -2,7 +2,7 @@ import ConfigureDestination from './_configure-destination.mdx';
 import ConfigureSource from './_configure-source.mdx';
 import ConfigureAuthenticateProvider from './_configure-authenticate-provider.mdx';
 import ConfigureStartSyncing from './_configure-start-syncing.mdx';
-import ConfigureSyncCommand from './_configure-sync-command.mdx';
```
Comment (erezrokah) [inline website/components/mdx/_configure-macOS.mdx:5]: Unrelated fix to make this filename more clear

File: website/pages/docs/quickstart/linux.mdx [hunk 3]
```diff
@@ -4,11 +4,11 @@ description: A quickstart guide for installing and running CloudQuery on Linux.
 ---
 
 import Intro from '../../../components/mdx/_intro.mdx'
-import SignIn from '../../../components/mdx/_cloudquery_sign_in.mdx'
```
Comment (erezrokah) [inline website/pages/docs/quickstart/linux.mdx:7]: Unrelated fix to make this filename to have consistent casing like other files

File: website/pages/docs/quickstart/macOS.mdx [hunk 4]
```diff
@@ -4,11 +4,11 @@ description: A quickstart guide for installing and running CloudQuery on MacOS.
 ---
 
 import Intro from '../../../components/mdx/_intro.mdx'
-import SignIn from '../../../components/mdx/_cloudquery_sign_in.mdx'
```
Comment (erezrokah) [inline website/pages/docs/quickstart/macOS.mdx:7]: Unrelated fix to make this filename to have consistent casing like other files

File: website/pages/docs/quickstart/windows.mdx [hunk 5]
```diff
@@ -4,11 +4,11 @@ description: A quickstart guide for installing and running CloudQuery on Windows
 ---
 
 import Intro from '../../../components/mdx/_intro.mdx'
-import SignIn from '../../../components/mdx/_cloudquery_sign_in.mdx'
```
Comment (erezrokah) [inline website/pages/docs/quickstart/windows.mdx:7]: Unrelated fix to make this filename to have consistent casing like other files

File: website/components/mdx/_cloudquery-sign-in.mdx [hunk 6]
```diff
@@ -8,9 +26,8 @@ You will be able to create a new account when you sign in with the CLI for the f
 
 To sign in from the CLI, run the following command:
 
-```bash
-cloudquery login
-```
+<SignIn os={props.os} />
```
Comment (erezrokah) [inline website/components/mdx/_cloudquery-sign-in.mdx:29]: This is the fix, pass the prop from each individual OS guide to the `SignIn` to render a different command

**Review by marianogappa — APPROVED** [review]
